frontend/internal-packages/agent/src/chat/chatProcessor.ts

```diff
@@ -3,7 +3,11 @@ import type { Repositories } from '../repositories'
 import { executeChatWorkflow } from './workflow'
 import type { WorkflowState } from './workflow/types'
 
-interface ChatProcessorParams {
+// Export for testing purposes
+export { executeChatWorkflow } from './workflow'
+export type { WorkflowState } from './workflow/types'
```

Comment:
Do I really need to re-export this? Can't we just `import { executeChatWorkflow } from '. /worflow'` at the caller?

Reply:
Thank you very much. I had omitted to check. I will take care.
https://github.com/liam-hq/liam/pull/1993/commits/a4b89d085e6021696e2451336817666ad6d29fa9